### ISSUE TITLE
Incompatibility of @pixi/node with @pixi/extract 7.2.4

### ISSUE THREAD

**Issue by UlyssesZh** [pixijs/node#11]
`@pixi/extract` 7.2.4 uses `ImageData` (introduced in commit [242eac](https://github.com/pixijs/pixijs/commit/242eac69b9d80a0ca5fce11dde396daf8cc13986)):

https://github.com/pixijs/pixijs/blob/v7.2.4/packages/extract/src/Extract.ts#L153

We need to polyfill this object correctly to the `ImageData` provided by node-canvas.

This bug is similar to #4 because they are both due to not polyfilling objects from node-canvas correctly in `@pixi/node`.

**Comment by bigtimebuddy** [comment]
Yes, extract doesn't work. Use the canvas directly. e.g., `app.renderer.view.toDataUrl('image/png')`. I updated the hello world to not use extract. Will likely remove extract plugin.

**Comment by UlyssesZh** [comment]
No need to remove that plugin, though. Wouldn't it be enough to modify that plugin so that it uses `ImageData` from node-canvas in Node.js environment?
